### ISSUE TITLE
Cannot set multiple preview size

### ISSUE THREAD

**Issue by auanasgheps** [rullzer/previewgenerator#203]
Hi,
I get an error when trying to use this command
`occ config:app:set --value="32 64 1024"  previewgenerator squareSizes`

However, single values work like:

`occ config:app:set --value="32"  previewgenerator squareSizes`
The value does not matter, as long as it's only one. 

See screenshot below. Can you help? 

![image](https://user-images.githubusercontent.com/20586878/91181757-9df3f200-e6e9-11ea-9d17-265d2a2422ee.png)

**Comment by auanasgheps** [comment]
Ok, I've found the right command

`sudo -u abc ./occ config:app:set --value="32 64 1024"  previewgenerator squareSizes`

abc is the root user in my container (linuxserver/nextcloud).

Fair enough.

**Comment by stavros-k** [comment]
Sorry for bringing this up, i'm also using linuxserver/nextcloud, i get the same error as your 1st post, but if u try as abc user, it says abc is not in the sudores file. 
Any idea?

**Comment by auanasgheps** [comment]
Honestly I don't know, what happens if you simply run
`sudo -u abc`
?
I use this prefix to run my commands in the nextcloud container.

**Comment by stavros-k** [comment]
this is what im getting, do you have the latest lsio docker?

```
root@fd46a71f4391:/# sudo -u abc occ
abc is not in the sudoers file.  This incident will be reported.
root@fd46a71f4391:/# sudo -u abc
usage: sudo -h | -K | -k | -V
usage: sudo -v [-AknS] [-g group] [-h host] [-p prompt] [-u user]
usage: sudo -l [-AknS] [-g group] [-h host] [-p prompt] [-U user] [-u user] [command]
usage: sudo [-AbEHknPS] [-C num] [-D directory] [-g group] [-h host] [-p prompt] [-R directory] [-T timeout] [-u user]
            [VAR=value] [-i|-s] [<command>]
usage: sudo -e [-AknS] [-C num] [-D directory] [-g group] [-h host] [-p prompt] [-R directory] [-T timeout] [-u user] file
            ...
root@fd46a71f4391:/# 

```

**Comment by auanasgheps** [comment]
Yes, I'm using the latest version, but I confirm `abc` is sudoer in my image. 

Docs [here ](https://github.com/linuxserver/docker-nextcloud#application-setup)mention `abc`, but says it is not needed anymore, at least for some commands.

**Comment by stavros-k** [comment]
Even tried pulling a new image and starting a new container. Still abc not in sudoers list.
anyway, thanks for the help! I'll see if i can sort it out on linuxserver/nextcloud repo.
